# Generated for NaoQi method ALLogger.logInStd
# Allows the logger to output logs to the standard output.

---


